shell/components/form/HookOption.vue

```diff
@@ -150,35 +150,33 @@ export default {
     <template v-if="selectHook === 'httpGet'">
       <h4>{{ t('workload.container.lifecycleHook.httpGet.title') }}</h4>
       <div class="var-row">
-        <template @update:value="update">
-          <LabeledInput
-            v-model:value="value.httpGet.host"
-            :label="t('workload.container.lifecycleHook.httpGet.host.label')"
-            :placeholder="t('workload.container.lifecycleHook.httpGet.host.placeholder')"
-            :mode="mode"
-          />
-          <LabeledInput
-            v-model:value="value.httpGet.path"
-            :label="t('workload.container.lifecycleHook.httpGet.path.label')"
-            :placeholder="t('workload.container.lifecycleHook.httpGet.path.placeholder')"
-            :mode="mode"
-          />
-          <LabeledInput
-            v-model:value.number="value.httpGet.port"
-            type="number"
-            :label="t('workload.container.lifecycleHook.httpGet.port.label')"
-            :placeholder="t('workload.container.lifecycleHook.httpGet.port.placeholder')"
-            :mode="mode"
-            required
-          />
-          <LabeledSelect
-            v-model:value="value.httpGet.scheme"
-            :label="t('workload.container.lifecycleHook.httpGet.scheme.label')"
-            :placeholder="t('workload.container.lifecycleHook.httpGet.scheme.placeholder')"
-            :options="schemeOptions"
-            :mode="mode"
-          />
-        </template>
+        <LabeledInput
+          v-model:value="value.httpGet.host"
+          :label="t('workload.container.lifecycleHook.httpGet.host.label')"
+          :placeholder="t('workload.container.lifecycleHook.httpGet.host.placeholder')"
+          :mode="mode"
+        />
```

Comment:
We discussed this OOB, but we may need to add `@update:value="update"` to each instance of `LabeledInput` that was previously wrapped by the template in order to avoid regressions. It needs to be confirmed if `update` was ever invoked when editing any of these instances of `LabeledInput` before doing so.

Reply:
I don't think it makes a difference in this case, but I've added @update:value="update" for consistency.